src/pybind/mgr/dashboard/module.py

```diff
@@ -220,11 +218,16 @@ class Module(MgrModule, CherryPyConfig):
 
     COMMANDS = [
         {
-            'cmd': 'dashboard set-session-expire '
+            'cmd': 'dashboard set-jwt-token-ttl '
                    'name=seconds,type=CephInt',
-            'desc': 'Set the session expire timeout',
+            'desc': 'Set the JWT token TTL in seconds',
             'perm': 'w'
         },
+        {
+            'cmd': 'dashboard get-jwt-token-ttl',
```

Comment:
Shouldn't we mention these commands (`set-jwt-token-ttl` and `get-jwt-token-ttl`) in the documentation?

I know, we didn't do it for `set-session-expire`, but for the sake of completeness I think it's worthwhile to document this configuration setting.

Reply:
I agree. Will take care of that